tailwind.config.js

```diff
@@ -19,6 +19,22 @@ export default {
     container: false, // Replaced with a custom container class
   },
   theme: {
+    screens: {
+      smd: { max: '639.98px' },
+      sm: '640px',
+
+      mdd: { max: '749.98px' },
+      md: '768px',
+
+      lgd: { max: '1023.98px' },
+      lg: '1024px',
+
+      xld: { max: '1279.98px' },
+      xl: '1280px',
+
+      '2xld': { max: '1535.98px' },
+      '2xl': '1536px',
```

Comment:
These are the default tailwind sizes, but also have a "down" variant that lets you select screens that are below a certain breakpoint's size. 

Really helps when styles deviate a lot on mobile.